README.md

```diff
@@ -15,6 +15,90 @@ for Deno.
 | [path](./path/)       | File path manipulation.                                         |
 | [testing](./testing/) | Testing                                                         |
 
+## Style Guide
+
+### Use the term "module" instead of "library" or "package"
+
+For clarity and consistency avoid the terms "library" and "package". Instead use
+"module" to refer to a single JS or TS file and also to refer to a directory of
+TS/JS code.
+
+### Use the filename "mod.ts" as the default entry point to a directory of code
+
+`index.ts` comes with the wrong connotations - and `main.ts` should be reserved
+for executable programs. The filename `mod.ts` follows Rust’s convention, is
+shorter than `index.ts`, and doesn’t come with any preconceived notions about
+how it might work.
+
+### Within `deno_std`, do not depend on external code
+
+`deno_std` is intended to be baseline functionality that all Deno programs can
+rely on. We want to guarantee to users that this code does not include
+potentially unreviewed third party code.
+
+### Within `deno_std`, minimize dependencies; do not make circular imports.
+
+Although `deno_std` is a standalone codebase, we must still be careful to keep
+the internal dependencies simple and manageable. In particular, be careful to
+not to introduce circular imports.
+
+### For consistency, use underscores, not dashes in filenames.
+
+Example: Instead of `file-server.ts` use `file_server.ts`.
+
+### Format code according using prettier.
+
+More specifically, code should be wrapped at 80 columns and use 2-space
+indentation and use camel-case.
```

Comment:
Suggest to use `format.ts` from root directory for this part?

Reply:
done